### ISSUE TITLE
🐛 Fails on long files

### ISSUE THREAD

**Issue by antonk52** [rome/tools#1221]
## Environment

```
$ rome rage --summary
Environment Variables
  Object {
    COLORFGBG: "7;0"
    COLORTERM: "truecolor"
    LANG: "en_US.UTF-8"
    TERM: "xterm-256color"
    TERM_PROGRAM: "iTerm.app"
    TMUX: "/private/tmp/tmux-1434521695/default,76660,1"
  }

User Config
  unset

Rome Version
  10.0.4-beta

Node Version
  12.16.1

Platform
  darwin x64 18.7.0

Terminal Features
  Object {
    background: "dark"
    colorDepth: 8
    columns: 106
    cursor: true
    hyperlinks: true
    isTTY: true
    unicode: true
  }

Client Flags
  Object {
    clientName: "cli"
    cwd: "/Users/antonk52/Documents/dev/personal/flowts"
    realCwd: "/Users/antonk52/Documents/dev/personal/flowts"
    silent: false
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 63_647_744
      pid: 98_253
      uptime: 0.676313229
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 63_647_744
        ownedBytes: 0
        ownedFileCount: 0
        pid: 98_253
        uptime: 0.67621736
      }
    ]
  }
```

## Steps To Reproduce

1. clone [flowts](https://github.com/zxbodya/flowts/) repository
2. install rome using `npm i -D rome`
3. init rome using `npx rome init` and see that it will complain with `Duplicate package name flowts`
4. Edit name to "flowts-cli" in `packages/cli/package.json`
5. Run check again with `npx rome check` and see it failing with the output below

```
ℹ Please open an issue with the details provided above if necessary
⚠ Running lint on packages/cli/src/rules/lib/react-dom.test.ts seems to be taking longer than expected.
  Have been waiting for 12s.
⚠ Running lint on packages/cli/src/rules/lib/bom.test.ts seems to be taking longer than expected. Have
  been waiting for 15s.
⚠ Worker 4 has not responded for 18s seconds. It is unlikely to become responsive. Currently processing:
- lint x 1
  - Object {
      parseOptions: Object {}
      options: Object {
        applySafeFixes: true
        compilerOptions: undefined
        prefetchedModuleSignatures: Object {}
        save: false
      }
      ref: Object {
        manifest: 3
        project: 2
        real: "/Users/antonk52/Documents/dev/personal/flowts/packages/cli/src/rules/lib/bom.test.ts"
        relative: "packages/cli/src/rules/lib/bom.test.ts"
        remote: false
        uid: "flowts/-cli/src/rules/lib/bom.test.ts"
      }
    }
ℹ Please open an issue with the details provided above if necessary
⚠ Running lint on packages/cli/src/rules/lib/dom.test.ts seems to be taking longer than expected. Have
  been waiting for 15s.
⚠ Running lint on packages/cli/src/rules/lib/react.test.ts seems to be taking longer than expected. Have
  been waiting for 3s.
⚠ Worker 2 has not responded for 19s seconds. It is unlikely to become responsive. Currently processing:
- lint x 1
  - Object {
      parseOptions: Object {}
      options: Object {
        applySafeFixes: true
        compilerOptions: undefined
        prefetchedModuleSignatures: Object {}
        save: false
      }
      ref: Object {
        manifest: 3
        project: 2
        real: "/Users/antonk52/Documents/dev/personal/flowts/packages/cli/src/rules/lib/dom.test.ts"
        relative: "packages/cli/src/rules/lib/dom.test.ts"
        remote: false
        uid: "flowts/-cli/src/rules/lib/dom.test.ts"
      }
    }
ℹ Please open an issue with the details provided above if necessary

 worker 2 project-rome/@internal/core/server/WorkerManager.ts:288:6 internalError/fatal  INTERNAL   FATAL
  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Did not respond for 19550ms and was checked 6 times

  1. <anonymous> (project-rome/@internal/core/server/WorkerManager.ts:288:6)
  2. Timeout.<anonymous> (project-rome/@internal/events/Bridge.ts:178:7)

  ⚠ Rome exited as this error could not be handled and resulted in a fatal error. Please report if
    necessary.


 project-rome/@internal/events/Bridge.ts:351:20 internalError/request ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Connection died

  1. ___R$project$rome$$internal$core$common$bridges$WorkerBridge_ts$default.end (
     project-rome/@internal/events/Bridge.ts:351:20)
  2. ___R$project$rome$$internal$core$server$WorkerManager_ts$default.end (
     project-rome/@internal/core/server/WorkerManager.ts:106:11)
  3. ___R$project$rome$$internal$core$server$Server_ts$default.onFatalError (
     project-rome/@internal/core/server/Server.ts:335:21)
  4. onFatalError (project-rome/@internal/core/server/WorkerManager.ts:287:17)
  5. Timeout.<anonymous> (project-rome/@internal/events/Bridge.ts:178:7)

  ℹ Error occurred while requesting lint for packages/cli/src/rules/lib/react.test.ts
```

## Expected Results

Rome should check the contents without failing and output the result to stdout.

**Comment by antonk52** [comment]
I've realized that I did not provide the link to the [repository](https://github.com/zxbodya/flowts/) 🤦‍♂️

**Comment by robert-chiniquy** [comment]
I'm being hit with this one too, Rome seems to be choking on a 13kLOC JSON file.

**Comment by ematipico** [comment]
This should not be the case anymore. Latest nightly should exit with error

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
